crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/panic_data.rs

```diff
@@ -10,6 +10,11 @@ pub fn try_extract_panic_data(err: &str) -> Option<Vec<Felt252>> {
     let re_string = Regex::new(r#"(?s)Execution failed\. Failure reason: "(.*?)"\."#)
         .expect("Could not create string panic_data matching regex");
 
+    let re_entry_point = Regex::new(
+        r"Entry point EntryPointSelector\((0x[a-fA-F0-9]{0,64})\) not found in contract\.",
+    )
+    .expect("Could not create entry point panic_data matching regex");
+
```

Comment:
Unfortunately, felt returned from cairoVM doesn't fulfill felt regex definition from spec and as a result doesn't capture the selector returned in error.
I don't think it will be a problem to left it in `(0x[a-fA-F0-9]{0,64})` from.

Reply:
It looks like cairoVM returns felts in padded form (that's why it's 64 chars after 0x, instead of 63 which is in spec). It's fine, let's leave it as it is 👍  but I think we should add a brief comment explaining why this regex is different that the spec's one.